addons/website_quote/models/sale_quote.py

```diff
@@ -55,21 +55,32 @@ def open_template(self):
 class SaleQuoteLine(models.Model):
     _name = "sale.quote.line"
     _description = "Quotation Template Lines"
-    _order = 'sequence, id'
+    _order = 'quote_id, sequence, id'
 
     sequence = fields.Integer('Sequence', help="Gives the sequence order when displaying a list of sale quote lines.",
         default=10)
     quote_id = fields.Many2one('sale.quote.template', 'Quotation Template Reference', required=True,
         ondelete='cascade', index=True)
     name = fields.Text('Description', required=True, translate=True)
-    product_id = fields.Many2one('product.product', 'Product', domain=[('sale_ok', '=', True)], required=True)
-    layout_category_id = fields.Many2one('sale.layout_category', string='Section')
+    product_id = fields.Many2one('product.product', 'Product', domain=[('sale_ok', '=', True)])
+    # QUESTION: what do I do with website_description for section and note?
     website_description = fields.Html('Line Description', related='product_id.product_tmpl_id.quote_description',
         translate=html_translate)
-    price_unit = fields.Float('Unit Price', required=True, digits=dp.get_precision('Product Price'))
+    price_unit = fields.Float('Unit Price', digits=dp.get_precision('Product Price'))
     discount = fields.Float('Discount (%)', digits=dp.get_precision('Discount'), default=0.0)
-    product_uom_qty = fields.Float('Quantity', required=True, digits=dp.get_precision('Product UoS'), default=1)
-    product_uom_id = fields.Many2one('uom.uom', 'Unit of Measure ', required=True)
+    product_uom_qty = fields.Float('Quantity', digits=dp.get_precision('Product UoS'), default=1)
+    product_uom_id = fields.Many2one('uom.uom', 'Unit of Measure')
+
+    line_type = fields.Selection(
```

Comment:
It's defined for the 3rd time.

Maybe we could think to define a mixin to add sections on any model that has a one2many. 

To be discussed, but not in the scope of this task.